client/layout/global-sidebar/footer.tsx

```diff
@@ -60,3 +61,23 @@ export const GlobalSidebarFooter: FC< {
 		</SidebarFooter>
 	);
 };
+
+export const GlobalSiteSidebarFooter: FC< {
+	selectedSiteSlug: string;
+	translate: LocalizeProps[ 'translate' ];
+} > = ( { selectedSiteSlug, translate } ) => {
+	return (
+		<SidebarFooter>
+			<a
+				href={ `https://${ selectedSiteSlug }/wp-admin/plugins.php` }
+				className="sidebar__footer-wpadmin tooltip tooltip-top"
+				title={ translate( 'WP Admin' ) }
+				data-tooltip={ translate( 'WP Admin' ) }
```

Comment:
It seems a bit duplicate. Maybe we don't need the tooltip here 🤔

![image](https://github.com/Automattic/wp-calypso/assets/13596067/4690ed11-9f32-4274-a659-0c021eef4292)

Reply:
Do you have any thoughts, referring to your previous review: https://github.com/Automattic/wp-calypso/pull/85013#discussion_r1426199874, @taipeicoder 🙂 

>I'm in favor of leaving out the shortcut, but keeping the tooltip for accessibility purposes. Even though it might be redundant 🙂